### PR TITLE
Add a mining(plasma) shotgun for 6,000 points with a gibtonite defuser for 1,000

### PR DESCRIPTION
**Pull request by redmoogle**
# General Documentation

### Intent of your Pull Request

Add a plasma cutter shotgun to the mining vendor for 6,000 points
Comes with a automatic gibtonite defuser for 1,000

The shotgun shoots 5 pellets that mine upto ~7 rocks each, 2 damage per shot and has 25 variance

The gibtonite defuser just instantly defuses the gibtonite when hit by it, thats all _because theres a mild chance you will blow up_

image

### Why is this change good for the game?

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Adds a plasma shotgun for 6,000
Adds a gibtonite defusing kit for 1,000

### What general grouping does this PR fall under? 
Mining _buff_

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
Uses 70 energy roughly 8-10 shots

# Changelog

:cl:  
rscadd: Adds a mining(plasma) shotgun for 6,000 points
rscadd: Adds a gibtonite defuser kit for 1,000 points
/:cl:

### CODE REVIEW

**Comment by UselessTheremin** [comment]
what does the defuser kit do exactly?

**Comment by UselessTheremin** [comment]
i mean how is it different than defusing gibtonite with a mining scanner

**Comment by redmoogle** [comment]
> 
> 
> what does the defuser kit do exactly?

so when mining with this shotgun, it has a habit of sometimes hitting the same tile twice (which can blow you up) the defuser kit just instantly defuses any gibtonite it hits preventing your imminent death

**Comment by maxion12345** [comment]
Now, gate it behind mining tech like all the other plasmacutters

**Comment by ilikesaying** [comment]
it's gated behind 6000 mining points

**Comment by ToasterBiome** [comment]
Cool idea but I think mining already has insane mining power with the satchel of holding and an advanced plasma cutter. I can't imagine being pwned by a plasma shotgun by accident when another miner isn't paying attention. I also thinking breaking a max of 35 blocks in one shot is kinda nutty even if you aren't gonna be destroying that many blocks in reality. I think if the plasma cutters are amazing for straight lines this should be good for wider shots. Maybe if the pellets mined like 2-3 blocks each and there was more spread? I don't know how much variance 25 is. is that 25 degrees?

I also think it should be a Protolathe item instead of a points reward. I associate utility items with the vending machines like the scanner and the epipens and more directly interactable things or upgradey things to the Protolathe. (because who buys the damn KA upgrades with mining points) I would assume this would be an "upgrade" and split from the regular plasma cutter. Maybe it could be wrapped into the research with the advanced one.

**Comment by Marmio64** [comment]
it shouldn't be gated behind research, it is a side grade to the advanced plasma cutter.

**Comment by ToasterBiome** [comment]
> it shouldn't be gated behind research, it is a side grade to the advanced plasma cutter.

...which is why it should be gated behind research. let me rephrase. gated behind the SAME research as the advanced plasma cutter. As in when you research Advanced Mining Technology you also get the shotgun.

**Comment by SomeguyManperson** [comment]
meatshot damage assuming all cutter shots hit is 10 making it only slightly more effective in direct combat than a normal cutter, and having some decent mining equipment behind points instead of materials might make people actually care about points a bit more

**Comment by ToasterBiome** [comment]
There's already decent mining equipment behind the points. Advanced Scanner is the one that comes to mind the most. The medipens are also the most useful thing you can buy. As I said before, it's utility stuff. They serve different functions

**Comment by redmoogle** [comment]
> There's already decent mining equipment behind the points. Advanced Scanner is the one that comes to mind the most. The medipens are also the most useful thing you can buy. As I said before, it's utility stuff. They serve different functions

Pickaxes

**Comment by ToasterBiome** [comment]
> > There's already decent mining equipment behind the points. Advanced Scanner is the one that comes to mind the most. The medipens are also the most useful thing you can buy. As I said before, it's utility stuff. They serve different functions
> 
> Pickaxes

I've actually never seen anyone buy a pickaxe with points because they're outclassed by literally everything from the Protolathe. Pickaxe buff when?

**Comment by redmoogle** [comment]
> 
> 
> > > There's already decent mining equipment behind the points. Advanced Scanner is the one that comes to mind the most. The medipens are also the most useful thing you can buy. As I said before, it's utility stuff. They serve different functions
> > 
> > 
> > Pickaxes
> 
> I've actually never seen anyone buy a pickaxe with points because they're outclassed by literally everything from the Protolathe. Pickaxe buff when?

pickaxes are just old remnants from the slow powercreep of ss13, we'd be fine with just the base version. or adding a mini-plasma cutter

**Review by Partheo — APPROVED** [review]
Looks sweet
